### PR TITLE
this is the the shop fragment update

### PR DESCRIPTION
**Pull request by shounakb95**

### CODE REVIEW

File: .idea/gradle.xml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="GradleMigrationSettings" migrationVersion="1" />
```
Comment (jasonn0118) [inline .idea/gradle.xml:3]: Can you explain why it has been removed?

File: .idea/misc.xml [hunk 2]
```diff
@@ -1,9 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (jasonn0118) [inline .idea/misc.xml:3]: Can you explain why it is changed?

File: .idea/vcs.xml [hunk 3]
```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$/.." vcs="Git" />
```
Comment (jasonn0118) [inline .idea/vcs.xml:4]: Please Explain this too

File: .idea/gradle.xml (hunk 1 above)
Comment (shounakb95) [reply .idea/gradle.xml:3]: i didn't do that i think it might have been my android studio

File: .idea/misc.xml (hunk 2 above)
Comment (shounakb95) [reply .idea/misc.xml:3]: i didn't do that i think it might have been my android studio

**Pull request merged**